src/engine/graph_node.h

```diff
@@ -50,7 +50,6 @@ enum GraphRenderFlags {
 #define GET_GRAPH_NODE_LAYER(flags)        ((flags & GRAPH_RENDER_LAYERS_MASK) >> GRAPH_RENDER_FLAGS_SIZE)
 #endif
 
-#ifdef DISABLE_GRAPH_NODE_TYPE_FUNCTIONAL
```

Comment:
Not sure I understand what needs to happen with this PR; is this a backwards removal or intentional?

Reply:
Intentional